includes/process-donation.php

```diff
@@ -230,7 +236,18 @@ function give_process_form_login() {
 	give_log_user_in( $user_data['user_id'], $user_data['user_login'], $user_data['user_pass'] );
 
 	if ( $is_ajax ) {
-		echo 'success';
+		$data['status'] = 'success';
```

Comment:
@raftaar1191 Can you use WordPress core function https://codex.wordpress.org/Function_Reference/wp_send_json_success